src/screens/HomePage/MapAccessories/MapAccessories.tsx

```diff
@@ -1,24 +1,30 @@
 import React from 'react';
-import { Row, Wrapper, AboutLink } from './MapAccessories.style';
+import { Row, Wrapper, AboutLink, TableWrapper } from './MapAccessories.style';
 import ShareButtons from 'components/SharedComponents/ShareButtons';
 import { EventCategory } from 'components/Analytics';
 import { getUrlAndShareQuote } from 'components/ShareBlock/ShareModelBlock';
+import { MapView } from '../utils';
 
 interface MapAccessoriesProps {
   renderThermometer: () => React.ReactElement;
   infoLink: string;
+  renderTable?: (locationScope: MapView) => React.ReactElement;
```

Comment:
FWIW- Since this component just automatically calls `renderTable(locationScope)` I think we could simplify this a bit and just have `table?: React.ReactElement` here, without needing `locationScope`

and then in MapBlock do:
```
          table={renderTable && renderTable(locationScope)}
```

We would only need to pass in `renderTable` if this component needed to potentially call it with different parameters depending on internal state or something.  The same actually goes for `renderThermometer`.  It could just be `thermometer: React.ReactElement` instead of being a function.